src/useScenes.ts

```diff
@@ -3,15 +3,109 @@ import { AppState, BinaryFiles } from "@excalidraw/excalidraw/types/types";
 import { ExcalidrawElement } from "@excalidraw/excalidraw/types/element/types";
 import { createScene } from "./creation";
 import { Drawing, Scene } from "./types";
-import { loadStorage, saveStorage } from "./persistence";
+import { loadStorage, saveStorage, SCENE_STORAGE_KEY } from "./persistence";
 import { isEqual } from "lodash";
+import { nanoid } from "nanoid";
 
 enum Initialisation {
   NotStarted,
   Started,
   Complete,
 }
 
+const emptyScene: Scene = {
```

Comment:
This doesn't seem very good.
- It feels too hacky.
- Changes aren't minimal.
- We need to change our code to follow Excalidraw changes.

I think what we should do instead is:
1. Delete all scenes except for the first one.
2. Clear drawing using Excalidraw API. (If it's not provided, let's give up for now.)